### PR TITLE
Patch 2

### PR DESCRIPTION
**Pull request by Magistone**
The 1st true entry is on line 1870

### CODE REVIEW

File: data/teams.json [hunk 1]
```diff
@@ -0,0 +1,185 @@
+[
+  {
+    "name": "Aftermath",
+    "abbrevation": "aM",
+    "players": [
+      "Hearttt",
+      "MbL",
+      "Nicov"
```
Comment (simonsan) [inline data/teams.json:8]: I'm thinking of a better way to represent players here. As this is kind of a common known term, but from databases we have the unique identifiers. Maybe to write a small script that enumerates the players in the players.yaml and put in a unique ID here would be good? Or what's your opinion about that @happyleavesaoc? Might be too complicated then to make PRs in the future and add new players. Hmm.

Comment (happyleavesaoc) [reply data/teams.json:8]: Ideally we use the `liquipedia` player property and the Liquipedia API to write a script that automatically generates this file.

**Comment by happyleavesaoc** [comment]
I'm going to merge this for now, but consider my comment about maintaining the teams file.

File: data/teams.json (hunk 1 above)
Comment (simonsan) [reply data/teams.json:8]: Good idea!

**Pull request merged**